examples/02-HFSS/Waveguide_Filter.py

```diff
@@ -0,0 +1,186 @@
+# Place irises from inner (highest integer) to outer, 1
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+for count in reversed(range(1, len(wgparams['w']) + 1)):
+    if count < len(wgparams['w']):  # Update zpos
+        zpos = zpos + "".join([" + l" + str(count) + " + "])[:-3]
+        iris = place_iris(zpos, "t", count)
+        iris = place_iris("-(" + zpos + ")", "-t", count)
+
+    else:  # Place first iris
+        zpos = zstart
+        iris = place_iris(zpos, "t", count)
+        if not is_even:
+            iris = place_iris("-(" + zpos + ")", "-t", count)
+
+###############################################################################
+# Draw the full waveguide with ports.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# parse_expr() is used to determine the numerical position
+# of the integration line endpoints on the waveguide port.
+
+
+var_mapping['port_extension'] = 1.5 * wgparams['l'][0]  # Used to evaluate expression with parse_expr()
+hfss['port_extension'] = str(var_mapping['port_extension']) + wgparams['units']
+hfss["wg_z_start"] = "-(" + zpos + ") - port_extension"
+hfss["wg_length"]  = "2*(" + zpos + " + port_extension )"
+wg_z_start = hfss.variable_manager["wg_z_start"]
+wg_length = hfss.variable_manager["wg_length"]
+hfss["u_start"] = "-a/2"
+hfss["u_end"]  = "a/2"
+hfss.modeler.create_box(["-b/2", "-a/2", "wg_z_start"], ["b", "a", "wg_length"],
+                        name="waveguide", matname="vacuum")
+
+###############################################################################
+# Use parse_expr() to evaluate the numerical Cartesian coordinates
+# needed to specify start and end points of the integration line
+# on the port surfaces.
+#
+wg_z = [wg_z_start.evaluated_value, hfss.value_with_units(wg_z_start.numeric_value + wg_length.numeric_value, "in")]
+
+count = 0
+ports = []
+for n, z in enumerate(wg_z):
+    face_id = hfss.modeler.get_faceid_from_position([0, 0, z], obj_name="waveguide")
+    u_start = [0, hfss.variable_manager["u_start"].evaluated_value,  z]
+    u_end = [0, hfss.variable_manager["u_end"].evaluated_value, z]
+
+    ports.append(hfss.create_wave_port(face_id, u_start, u_end, portname="P" + str(n + 1), renorm=False))
+
+#################################################################################
+#  See pyaedt.modules.SetupTemplates.SolveSweeps.SetupNames
+#  for allowed values for setuptype.
+
+setup = hfss.create_setup("Setup1", setuptype="HFSSDriven",
+                           MultipleAdaptiveFreqsSetup=['9.8GHz', '10.2GHz'],
+                           MaximumPasses=5)
+
+setup.create_frequency_sweep(
+    unit="GHz",
+    sweepname="Sweep1",
+    freqstart=9.5,
+    freqstop=10.5,
+    sweep_type="Interpolating",
+    )
+
+
+#################################################################################
+#  Solve the project with 2 tasks.
+#  Each frequency point will be solved simultaneously.
```

Comment:
```suggestion
#  Solve the project with two tasks.
#  Each frequency point is solved simultaneously.
```
Google style guidelines say to avoid future verb tense.